### ISSUE TITLE
Allow bots to un/schedule all other bots.

### ISSUE THREAD

**Issue by iammattmartin** [jstanden/cerb#673]
One limitation now appears you can only make some bot behaviours interact with other behaviours on the same bot.

We have split ours bots up to things like SLA, webhooks, timeouts etc. On one or two, when there is another action (i.e close), we want ALL bots/behaviours removed. We can't do that when they don't share a common link.

For now, we've had to code into behaviours more checks to stop things happening, like sending a SLA reminder for a then closed ticket.

It would be immensely helpful for this to be possible without having to consider factoring in HTTP calls, more webhooks or other hacky ways etc.

**Comment by jstanden** [comment]
Since scheduled behaviors are a first class record now (profiles, cards, etc), you can use the 'Record' actions in bots to create, modify, or delete them.

Here's an example behavior that finds the scheduled behaviors on task #123 and deletes them:

~~~
{
    "behavior": {
        "uid": "behavior_907",
        "title": "Remove scheduled behaviors on target",
        "is_disabled": false,
        "is_private": true,
        "priority": 50,
        "event": {
            "key": "event.macro.bot",
            "label": "Custom behavior on bot"
        },
        "nodes": [
            {
                "type": "action",
                "title": "Find scheduled behaviors on target",
                "status": "live",
                "params": {
                    "actions": [
                        {
                            "action": "core.bot.action.record.search",
                            "context": "scheduled_behavior",
                            "query": "on.task:(id:123)",
                            "expand": "",
                            "object_placeholder": "_records"
                        }
                    ]
                }
            },
            {
                "type": "loop",
                "title": "Iterate scheduled behaviors",
                "status": "live",
                "params": {
                    "foreach_json": "{{_records|json_encode}}",
                    "as_placeholder": "scheduled_behavior"
                },
                "nodes": [
                    {
                        "type": "action",
                        "title": "Delete",
                        "status": "live",
                        "params": {
                            "actions": [
                                {
                                    "action": "core.bot.action.record.delete",
                                    "context": "scheduled_behavior",
                                    "id": "{{scheduled_behavior.id}}",
                                    "run_in_simulator": "0"
                                }
                            ]
                        }
                    }
                ]
            }
        ]
    }
}
~~~

You could modify that so it only deletes scheduled behaviors of a certain type, etc.

Assuming your bot has permissions (e.g. app-owned), you can create cross-bot scheduled behaviors the same way:

<img width="611" alt="screen shot 2018-06-15 at 11 45 56 am" src="https://user-images.githubusercontent.com/63288/41484445-b8477f6c-7091-11e8-9bce-77a4787370c8.png">

**Comment by jstanden** [comment]
Whoops. That requires #634, which was part of early 8.4 development, which merged into 9.0.  It's available in 9.0-beta, but will be released shortly.

That's still going to be the better approach for what you're asking for.  It gives you full control over the automation.
